Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.7-buster as base
+
+WORKDIR /srv
+#Update the system and install additional package
+RUN apt-get update && apt-get install -y curl wget
+COPY poetry.toml poetry.lock pyproject.toml /srv/
+RUN pip install poetry && poetry config virtualenvs.create false && poetry install
```

Comment:
Really minor, but you could `pip install poetry` before copying anything in so there's even less to do when a dependency changes.